stdlib/2.7/bisect.pyi

```diff
@@ -0,0 +1,17 @@
+def bisect(a,x,lo=0,hi=0): # type: (list,object,int,int) -> int
```

Comment:
Same as in the other file, using PEP 3107 for this is preferred:

def bisect(a:list, x, lo:int=..., hi:int=...) -> int: ...